packages/discovery-react-components/src/utils/document/documentUtils.ts

```diff
@@ -44,6 +45,7 @@ export function findOffsetInDOM(
 
   do {
     endNode = allNodes[idx];
+    endNode.normalize();
```

Comment:
The 2 calls shouldn't be here: we'd be calling `normalize()` unnecessarily, on nodes that we don't care about. We only need this in `getTextNodeAndOffset`